README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Engine and docker-compose
+  * Mac:
+    1. Install the [Docker Platform](https://www.docker.com/products/overview#/install_the_platform).
```

Comment:
Docker sure changes fast... They released an "Enterprise Edition" and rebranded the regular one to "Community Edition" and changed the links. https://blog.docker.com/2017/03/docker-enterprise-edition/